content/en/serverless/azure_app_services.md

```diff
@@ -57,17 +59,82 @@ The Datadog .NET APM extension supports the following .NET runtimes in both x64
 8. Restart the main application: click **Stop**, wait for a full stop, then click **Start**.
     {{< img src="infrastructure/serverless/azure_app_services/restart.png" alt="Stop and restart page" >}}
 
-### Logging from Azure Web Apps
-
-Logs for Azure Web Apps can be submitted to Datadog with Eventhub using the process described in the [Azure Integration documentation][8]. **Note**: The Eventhub must be located in the same region as your web application.
-
-Once the Eventhub and forwarder function are configured, create a diagnostic setting for your web application. Select the logs you want to send to Datadog:
-
-{{< img src="serverless/azure_diagnostics.png" alt="Diagnostics Settings" >}}
-
-Once you establish the logging pipeline for your application, Trace ID injection allows you to [connect logs and traces][9] in Datadog. To enable this with the extension, add an application setting `DD_LOGS_INJECTION:true`.
-
-**Note**: Trace ID injection occurs in the application, so application logs include the trace ID. Other categories of [diagnostic logs available from Azure][10], like HTTP logs and audit logs, do not include the trace ID.
+### Log collection
+
+#### Serilog
+
```

Comment:
Before we dive into the instructions, let's add an introduction to the process and include the bit about the trace ID correlation setting:

Sending logs from your application in Azure App Services to Datadog requires the use of Serilog. Submitting logs with this method also allows for Trace ID injection, which makes it possible to connect logs and traces in Datadog. To enable Trace ID injection this with the extension, add an application setting `DD_LOGS_INJECTION:true`."

Note: since this occurs inside your application, any Azure Platform logs you may be submitting via diagnostic settings will not include the Trace ID.